src/components/AvatarGroup/SbAvatarGroup.vue

```diff
@@ -102,8 +102,11 @@ export default {
       }
     },
 
-    handleMoreAvatarsClick(event, vm) {
-      this.$emit('click', event, vm)
+    handleMoreAvatarsClick() {
+      this.isVisibleDropdown = !this.isVisibleDropdown
+      this.$emit('toggle-visible-dropdown', {
```

Comment:
I changed the event name because it was called 'click' which is usually considered a reserved namespace as it is a default browser event.